### PR TITLE
Adds list of samples to android-testing README

### PR DESCRIPTION
**Pull request by JoseAlcerreca**

### CODE REVIEW

**Comment by bootstraponline** [comment]
shouldn't this be merged to master?

File: README.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
-
 A collection of samples demonstrating different frameworks and techniques for automated testing.
 
+### Espresso Samples
+
+**[BasicSample](https://github.com/googlesamples/android-testing/blob/master/ui/espresso/BasicSample)** - Basic Espresso sample
+
+**[CustomMatcherSample](https://github.com/googlesamples/android-testing/blob/master/ui/espresso/CustomMatcherSample)** - Shows how to extend Espresso to match the *hint* property of n EditText
+
+**[DataAdapterSample](https://github.com/googlesamples/android-testing/blob/master/ui/espresso/DataAdapterSample)** - Showcases the `onData()` entry point for Espresso, for lists and dapterViews
```
Comment (bootstraponline) [inline README.md]: what is a `dapterView`?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,43 @@
...
 ===================================
 
-
 A collection of samples demonstrating different frameworks and techniques for automated testing.
 
+### Espresso Samples
+
+**[BasicSample](https://github.com/googlesamples/android-testing/blob/master/ui/espresso/BasicSample)** - Basic Espresso sample
+
+**[CustomMatcherSample](https://github.com/googlesamples/android-testing/blob/master/ui/espresso/CustomMatcherSample)** - Shows how to extend Espresso to match the *hint* property of n EditText
```
Comment (bootstraponline) [inline README.md]: `n EditText`

Comment (JoseAlcerreca) [inline README.md]: Addressed. Thanks.

**Pull request merged**